trin-core/src/portalnet/overlay.rs

```diff
@@ -294,26 +294,24 @@ where
 
         // HashMap to temporarily store all interested ENRs and the content.
         // Key is base64 string of node's ENR.
-        let mut enrs_and_content: HashMap<String, Vec<RawContentKey>> = HashMap::new();
+        let mut enrs_and_content: HashMap<String, Vec<(RawContentKey, ByteList)>> = HashMap::new();
 
         // Filter all nodes from overlay routing table where XOR_distance(content_id, nodeId) < node radius
-        for content_key_value in content {
+        for content_item in content {
```

Comment:
*nit* `for (content_key, content_value) in content` could get rid of the `0` and `1` indexing further down